packages/next-templates/src/app/card-component/card/page.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { Heading } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import Image from 'next/image';
+import React, { FC, HTMLAttributes } from 'react';
+import './index.style.css';
+
+interface CardListItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'children'> {
+  headerlevel: number;
+  title?: string;
+  preheading?: string;
+  children?: string;
+  image?: {
+    url: string;
+    alt: string;
+  };
+  link: {
+    href: string;
+  };
+
+  // do we want an article around the card content. Default is no
+  useArticle?: boolean | false;
+}
+
+export const CardListItem: FC<CardListItemProps> = ({
```

Comment:
Using `FC` has several downsides. I would suggest to refactor it to the fashion all the Utrecht components are declared:

```js
export const Component = ({ arg }: ArgProps) => (<li>...</li>);
```

Or:

```js
export const Component = ({ arg }: PropsWithChildren<ArgProps>) => (<li>...</li>);
```